src/RequestService/RequestService.ts

```diff
@@ -0,0 +1,98 @@
+
+enum METHODS  {
+    GET = 'GET',
+    POST = 'POST',
+    PUT = 'PUT',
+    DELETE = 'DELETE',
+    PATCH = 'PATCH'
+};
+
+
+function queryStringify(data: string) {
+    let params: string[] = [];
+    
+    Object.entries(data).forEach(([key, value]: [string, string]) => {
+        params.push(`${key}=${value}`);
+    })
+
+    return `?${params.join('&')}`
+}
+
+type Options = {
+    data: any,
+    method: string,
+    headers: string,
+    timeout: number
```

Comment:
Тут можно типизировать детальнее
```suggestion
    data: unknown,
    method: METHODS,
    headers: Record<string, string>,
    timeout: number
```